_includes/footer_scripts.html

```diff
@@ -1,8 +1,15 @@
-<script src="{{ site.baseurl }}/assets/mobile-menu.js"></script>
-<script src="{{ site.baseurl }}/assets/scroll-to-anchor.js"></script>
-<script src="{{ site.baseurl }}/assets/external-links-new-tab.js"></script>
-<script src="{{ site.baseurl }}/assets/search-bar.js"></script>
-<script src="{{ site.baseurl }}/assets/cookie-banner.js"></script>
+{% if page.title == 'Deep Learning with PyTorch' %}
+  <script src="{{ site.baseurl }}/assets/mobile-menu.js"></script>
+  <script src="{{ site.baseurl }}/assets/scroll-to-anchor.js"></script>
+  <script src="{{ site.baseurl }}/assets/external-links-new-tab.js"></script>
+  <script src="{{ site.baseurl }}/assets/cookie-banner.js"></script>
+{% else %}
+  <script src="{{ site.baseurl }}/assets/mobile-menu.js"></script>
+  <script src="{{ site.baseurl }}/assets/scroll-to-anchor.js"></script>
+  <script src="{{ site.baseurl }}/assets/external-links-new-tab.js"></script>
+  <script src="{{ site.baseurl }}/assets/search-bar.js"></script>
+  <script src="{{ site.baseurl }}/assets/cookie-banner.js"></script>
+{% endif %}
```

Comment:
Instead of repeating all of the scripts here in your conditional, you should think about just targeting `search-bar.js` since that's where the console error comes from--something like: if page.deep_learning == false then load the script.